### ISSUE TITLE
Invoice Payment Status

### ISSUE THREAD

**Issue by hobejeans** [YetiForceCompany/YetiForceCRM#13414]
I have gone through all the threads that has to do with invoicing and I can not find any information that explains how to close an invoice. The payment status is greyed out and i can not update the status.
Please explain how to do this. Thank you

**Comment by vdpawel** [comment]
#13385

**Comment by hobejeans** [comment]
Thanks for responding... I created a **payments in** record for the invoice & it is still showing "Not Paid."
Am I missing something?

**Comment by hobejeans** [comment]
![Screen Shot 2020-08-22 at 1 34 25 PM](https://user-images.githubusercontent.com/45261773/90962204-5f0d3480-e47c-11ea-8e78-7d4d9970c89d.png)

The same grey areas are un-editable when a Payment-In  record is created for an invoice

**Comment by vovpff** [comment]
As I said you can search for solution.
Check this #13105

**Comment by hobejeans** [comment]
I figured out the problem. CRON was not set up correctly. I configured the settings according to the instructions [here](https://yetiforce.com/en/component/zoo/item/enable-cron)... 
The instructions above also helped with changing from a gray (created) status to paid or denied status. Without the cron being fixed it did not update in invoice or sales order until cron was configured properly. It takes a couple of minutes for the status changes to be updated so be patient.
